### ISSUE TITLE
Author creation/deletion buttons have no hover hint

### ISSUE THREAD

**Issue by Some-Birb7190** [lilopkins/evidenceangel#114]
For accessibility, buttons that do not have their action written next to them should have hover hints describing what they do.
The 'add author' and 'delete author' buttons do not have hover hints
